template/server/messaging.py

```diff
@@ -274,9 +296,6 @@ async def execute(
         env_vars: Dict[StrictStr, str],
         access_token: str,
     ):
-        message_id = str(uuid.uuid4())
```

Comment:
moved the logic in the retry loop, so we can safely prevent mixing the executions